task5/src/main/java/com/kwitter/GetServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.kwitter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/get")
+public class GetServlet extends HttpServlet {
+
+    final String lengthLimitExceededText = "length of name more than 100 symbols";
```

Comment:
Please declare all constants as `private static final String` (or ...int, etc) `NAME_OF_CONSTANT_IN_UPPERCASE`. This is by the strict code convention